vue-thacer/src/assets/js/thacer-map-create-layer.js

```diff
@@ -211,7 +211,7 @@ export function createMarkerClusterGroupChronique() {
 }
 
 // -----------------------------------EfA Geoserver WMS ---------------------------------------------
-export function createTileLayerSIG_thasos() {
+export function createTileLayersigThasos() {
```

Comment:
Désolé, mais ça devrait pas être `createTileLayerSigThasos`

Reply:
@AlZorba à propos, pas hyper interressant : https://stackoverflow.com/questions/15526107/acronyms-in-camelcase

Reply:
yes